views/navbar.erb

```diff
@@ -0,0 +1,5 @@
+<ul style="list-style-type:none", padding-left: 0;>
+  <li><%= erb(:button, :locals => { :action => "signup", :name => "Sign Up", :method => nil, :value => nil }) %></li>
+  <li><%= erb(:button, :locals => { :action => "signin", :name => "Sign In", :method => nil, :value => nil  }) %></li>
+  <li><%= erb(:button, :locals => { :action => "profile", :name => "Profile", :method => nil, :value => nil  }) %></li>
+</ul>
```

Comment:
It's smart the idea to have a navbar and store in there all your buttons you need on your multiple views. It helps you to keep clean your project.